SU-SWS/stanford_profile
# D8 core 1431

# READY FOR REVIEW

# Summary
- Validation that the url entered points to the site being used

# Needed By (Date)
- June 8th

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Comment out Test for stanford domain if needed on local
3. navigate to admin/config/system/basic-site-settings
4. verify you can only enter the base url of the site

# Affected Projects or Products
- All stacks implementing this change

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/D8CORE-1431

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
